src/components/dialogs/create-study-dialog.js

```diff
@@ -151,14 +158,39 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
     const [folderSelectorOpen, setFolderSelectorOpen] = useState(false);
     const [activeDirectoryName, setActiveDirectoryName] = useState(null);
 
+    const [formatWithParameters, setFormatWithParameters] = useState([]);
+
     const [triggerReset, setTriggerReset] = React.useState(true);
 
+    const [isParamsDisplayed, setIsParamsDisplayed] = useState(false);
+
+    const handleShowParametersClick = () => {
+        setIsParamsDisplayed((oldValue) => !oldValue);
+    };
+
+    const metasAsArray = formatWithParameters?.parameters || [];
```

Comment:
Why we do not use directly the state  formatWithParameters with the final value (parameters or [])

Reply:
fixed, I agree it makes the component easier to read